flash/image/segmentation/serialization.py

```diff
@@ -90,7 +90,7 @@ def _visualize(self, labels):
         plt.imshow(labels_vis)
         plt.show()
 
-    def serialize(self, sample: Dict[str, torch.Tensor]) -> torch.Tensor:
+    def transform(self, sample: Dict[str, torch.Tensor]) -> torch.Tensor:
```

Comment:
Maybe, although it definitely shouldn't be called serialize any more. As the role of `Output` is to convert raw predictions into the desired output format I think transform is fine. But it could otherwise be something more explicit like `convert(raw_predictions)` rather than `transform(sample)`

Reply:
I was thinking about `export_transform`.